src/components/ui/areaCardAssessment.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import {
+  Sheet,
+  SheetContent,
+  SheetDescription,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from "~/components/ui/sheet";
+import { Card, CardContent, CardHeader, CardTitle } from "./card";
+import { Button } from "./button";
+import { Icons } from "../icons";
+import { Badge } from "~/components/ui/badge";
+import { Separator } from "~/components/ui/separator";
+
+export interface AreaProps {
+  id: string;
+  name: string;
+  visible?: boolean;
+  toggleVisibility?: () => void;
+}
+
+const AreaCardAssessment = React.forwardRef<HTMLDivElement, AreaProps>(
+  (area, ref) => {
+    return (
+      <Card
+        title={area.name}
+        className="relative flex h-full w-full flex-col items-center justify-center"
+        ref={ref}
+      >
+        <div className="absolute right-2 top-2 space-x-1">
+          <Button
+            variant="outline"
+            size="icon"
+            className="h-4 w-4 shrink-0 rounded-full opacity-50 hover:bg-accent hover:text-accent-foreground"
+            onClick={area.toggleVisibility}
+          >
+            {area.visible ? <Icons.show /> : <Icons.hide />}
+          </Button>
+
+          {/* Info icon to trigger info sheet */}
+          <SheetTrigger>
+            <Button
+              variant="outline"
+              size="icon"
+              className="h-4 w-4 shrink-0 rounded-full opacity-50 hover:bg-accent hover:text-accent-foreground"
+            >
+              <Icons.info />
+            </Button>
+          </SheetTrigger>
+
+          {/* Edit icon to trigger assess sheet */}
+          <Sheet>
+            <SheetTrigger>
```

Comment:
SheetTrigger is already a button, putting a button inside of a button creates Hydration errors.

Reply:
Will remove the nested button.

One thing I noticed: the button of `<Button>` component looks different from the button of `<SheetTrigger>`, see screenshots here (the button of `<Button>` component has a slightly gray background)
![image](https://github.com/user-attachments/assets/db96d1bc-d644-4787-9295-b215c0994f7d)
If a button inside of sheettrigger, they look the same
![image](https://github.com/user-attachments/assets/2635436c-0a88-4ef9-a888-18be8aa0f690)

But of course, will remove button inside of sheettrigger, thank you for point it out